### PR TITLE
Update License and CoC files

### PR DESCRIPTION
**Pull request by kinoppyd**
To follow the SmartHR OSS Guideline, add license and CoC files.
[smarthr-oss-guideline/GUIDELINE.jp.md](https://github.com/kufu/smarthr-oss-guideline/blob/main/GUIDELINE.jp.md)

### CODE REVIEW

**Comment by mkmn** [comment]
🙏 
https://github.com/kufu/tsubaki#license

**Review by AtsushiM — APPROVED** [review]
👍

**Comment by kinoppyd** [comment]
@mkmn thanks, i'd fix it.

File: README.md [hunk 1]
```diff
@@ -125,5 +125,5 @@ Everyone interacting in the tsubaki project’s codebases, issue trackers, chat
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+The gem is available as open source under the terms of the [MIT License](https://github.com/kufu/tsubaki/blob/master/LICENSE.md)).
```
Comment (mkmn) [inline README.md:128]:
```suggestion
The gem is available as open source under the terms of the [MIT License](https://github.com/kufu/tsubaki/blob/master/LICENSE.txt)).
```

**Pull request merged**